<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreLocation</class>
 <widget class="QWidget" name="FenetreLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transports à la carte</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QPushButton" name="retour">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="choixVehicule">
     <widget class="QPushButton" name="chooseButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Choisir</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listVehicule">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>281</width>
        <height>111</height>
       </rect>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="busButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bus</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="carButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Voiture</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="bikeButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vélo</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="details">
     <widget class="QLabel" name="titrePage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fiche détaillée du véhicule</string>
      </property>
     </widget>
     <widget class="QLabel" name="fiche">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>231</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="validationChoix">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
